test/integration/patients/worklist/schedule.js

```diff
@@ -1498,4 +1695,51 @@ context('schedule page', function() {
       .should('contain', 'Test Patient')
       .should('contain', 'Second Action');
   });
+
+  specify('clinician in no groups', function() {
+    cy
+      .routeCurrentClinician(fx => {
+        fx.data.relationships.groups.data = [];
+        return fx;
+      })
+      .routeGroupsBootstrap()
+      .routeDirectories(fx => {
+        fx.data = [{
+          attributes: {
+            name: 'Insurance Plans',
+            slug: 'insurance',
+            value: [
+              'BCBS PPO 100',
+              'Medicare',
+            ],
+          },
+        }];
+
+        return fx;
+      })
+      .routeActions()
+      .visit('/schedule')
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('not.contain', 'filter[group]');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-group-filter-region]')
+      .should('be.empty');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .click();
+
+    cy
+      .get('[data-sidebar-region]')
+      .find('[data-filter-button]')
+      .should('have.length', 1)
+      .first()
+      .get('.sidebar__label')
+      .should('contain', 'Insurance Plans');
+  });
```

Comment:
This loads the schedule where the clinician has no groups. And verifies that the groups filter is not shown in the filters section above the list or the filters sidebar.